src/main/webapp/app/admin/dashboard/admin-dashboard.component.ts

```diff
@@ -0,0 +1,86 @@
+import { defineComponent, inject, ref, type Ref } from 'vue';
+
+import type { ILocation } from '@/shared/model/location.model';
+import LocationService from '@/entities/location/location.service';
+import type { IAdminChecklist } from '@/shared/model/admin-checklist.model';
+import AdminDashboardService from '@/admin/dashboard/admin-dashboard.service';
+import axios from 'axios';
+
+export default defineComponent({
+  compatConfig: { MODE: 3 },
+  name: 'AdminDashboard',
+  setup() {
+    const locationService = inject('locationService', () => new LocationService());
+    const adminDashboardService = inject('adminDashboardService', () => new AdminDashboardService());
+
+    const locations: Ref<ILocation[]> = ref([]);
+    const checklists: Ref<IAdminChecklist[]> = ref([]);
+    const bccForAllUsers: Ref<string> = ref('');
+    const countAllProfiles: Ref<number> = ref(0);
+    const bccForAllUsersWithUncompleteProfile: Ref<string> = ref('');
+    const countUncompleteProfiles: Ref<number> = ref(0);
+
+    const initRelationships = () => {
+      locationService()
+        .retrieve()
+        .then((res: { data: ILocation[] }) => {
+          locations.value = res.data;
+        });
+
+      adminDashboardService()
+        .checklist()
+        .then((res: { data: IAdminChecklist[] }) => {
+          checklists.value = res.data;
+          const ListOfUsersWithUncompleteProfile: string[] = [];
+          const ListOfUsers: string[] = [];
+
+          res.checklist.forEach((c: IAdminChecklist) => {
+            if (!c.mandatoryComplete && c.mail != null) {
+              ListOfUsersWithUncompleteProfile.push(c.mail);
+            }
+
+            if (c.mail != null) {
+              ListOfUsers.push(c.mail);
+            }
+          });
+          bccForAllUsers.value = 'mailTo:?bcc=' + ListOfUsers.join(';') + '&subject=Vollständiges Profil&body=Sehr geehrte Leuts';
```

Comment:
This is not a German salutation. The correct formal way is "Sehr geehrte Damen und Herren". However, in the other e-mails we use the more informal salutation "Moin!". Please change it accordingly.